analyzer/tests/functional/ctu_failure/test_ctu_failure.py

```diff
@@ -54,8 +55,13 @@ def __set_up_test_dir(self, project_path):
 
         self.ctu_has_analyzer_display_ctu_progress = \
             host_check.has_analyzer_feature(self.__getClangSaPath(),
-                                            '-analyzer-display-ctu-progress')
-        print("Has -analyzer-display-ctu-progress? " +
+                                            ['-Xclang',
+                                             '-analyzer-config',
+                                             '-Xclang',
+                                             'display-ctu-progress=true'],
+                                            self.env)
+
+        print("Has display-ctu-progress=true? " +
```

Comment:
Please remove this debug message when you are done.

Reply:
This was previously there too, I've just renamed the config value which changed.